src/main/webapp/account.jsp

```diff
@@ -60,8 +60,10 @@
                 </div>
                 <br/>
                 <input type="text" name="phone-input" id="phone-input" required="true" placeholder="Input your phone number here:">
+                <input type="hidden" name="lat" id="lat-input">
+                <input type="hidden" name="lng" id="lng-input">
                 <br/><br/>
-                <button type="submit" id="submit-button"/>GET STARTED</button>
+                <button type="submit" id="submit-button" title="You must mark your personal address on the map" disabled>GET STARTED</button>
```

Comment:
As discussed with @LeonardZ0731 , he adding a required red star next to the label on top of the map on his PR #88 to emphasize the fact that this is a required part of the form.